GitTfs/Util/ChangeSieve.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
```

Comment:
This line is no more needed since you remove the annotation (Sorry, I didn't see that earlier)